src/components/utils/loadFonts.ts

```diff
@@ -4,9 +4,10 @@ export function loadFonts() {
     return;
   }
   appendedFonts = true;
-  const font = document.createElement('link');
-  font.href =
-    'https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600;700&display=swap';
-  font.rel = 'stylesheet';
-  document.head.appendChild(font);
+  for (const family of ['Inter', 'Roboto Mono']) {
+    const font = document.createElement('link');
+    font.href = `https://fonts.googleapis.com/css2?family=${family}:wght@400;500;600;700&display=swap`;
```

Comment:
can we avoid loading fonts from google? let us host them. 
Is this the way how to add fonts? can't that be done in CSS?

Reply:
Also we need to replace with Open Runde. Where can we import them from, https://getalby.com?

It looks like we can use https://developer.mozilla.org/en-US/docs/Web/API/CSS_Font_Loading_API instead of injecting into the head

Reply:
Moved to separate issue: https://github.com/getAlby/lightning-wallet-connect/issues/31